rc/pdfpcrc

```diff
@@ -1,129 +1,123 @@
+bind g                  goto
+
+# Bookmarking [the current action names are misleading]
+bind m                  lastSlide
+bind S+m                jumpLastSlide
+
+# History? [these 3 are seriously broken]
+bind S+BackSpace        histBack
+bind S+End              nextUnseen
+bind S+Home             prevSeen
+
+# The overview mode
+bind Tab                overview
+
+# Toolbox
+bind t                  toggleToolbox
 
 # Exit
-bind Escape    quit
-bind q         quit
+bind C+q                quit
+
+# Presentation state
+bind b                  blank
+bind f                  freeze
+bind h                  hide
+bind Escape             exitState
 
 # Timer related
-bind s         start
-bind p         pause
-bind r         resetTimer
-bind R         reset
-
-# Change state
-bind b         blank
-bind period    blank
-bind f         freeze
-bind h         hide
-
-# Presentation properties
-bind n         note
-bind o         overlay
-bind e         endSlide
-bind S+s       lastSlide
-bind S+r       jumpLastSlide
+bind s                  start
+bind p                  pause
+bind C+t                resetTimer
 
-# Annotation modes
-bind 0         switchMode normal
-bind KP_0      switchMode normal
-bind 2         switchMode eraser
-bind KP_2      switchMode eraser
-bind 5         switchMode pen
-bind KP_5      switchMode pen
-bind 8         switchMode pointer
-bind KP_8      switchMode pointer
+# Presentation editing
+bind C+n                note
+bind C+o                overlay
+bind C+e                endSlide
 
-# Toolbox
-bind t toggleToolbox
-
-# Font scaling
-bind plus      increaseFontSize
-bind equal     increaseFontSize
-bind KP_Add    increaseFontSize
-bind bracketleft increaseFontSize
-bind minus     decreaseFontSize
-bind KP_Subtract decreaseFontSize
-bind bracketright decreaseFontSize
-
-# Pointer
-bind 7         decreasePointer
-bind KP_7      decreasePointer
-bind 9         increasePointer
-bind KP_9      increasePointer
-
-# Drawing
-bind 6         clearDrawing
-bind KP_6      clearDrawing
-bind 4         toggleDrawings
-bind KP_4      toggleDrawings
-
-bind 1         decreasePen
-bind KP_1      decreasePen
-bind 3         increasePen
-bind KP_3      increasePen
+# Reset the presentation
+bind C+z                reset
+
+# Annotation modes
+bind 1                  switchMode normal
+bind KP_1               switchMode normal
+bind 2                  switchMode pointer
+bind KP_2               switchMode pointer
+bind 3                  switchMode pen
+bind KP_3               switchMode pen
+bind 4                  switchMode eraser
+bind KP_4               switchMode eraser
+
+# Mode-sensitive size adjustment (notes|pointer|pen|eraser)
+bind plus               increaseSize
+bind equal              increaseSize
+bind KP_Add             increaseSize
+bind minus              decreaseSize
+bind KP_Subtract        decreaseSize
+
+# Drawings
+bind c                  clearDrawing
+bind d                  toggleDrawings
+
+# Pen colors
```

Comment:
We are loosing all Shift-Number combinations just with switching colors. Do we need all?

Reply:
Yes, I thought about it, too. Lately, I use the toolbox to change colors (and I do it infrequently anyway). But probably there are hard-die folks who prefer everything keyboard. Unfortunately, we cannot run a poll among users to get a representative statistics of opinions. Do you have an idea what these combinations could be used for, if we drop their current bindings?

Reply:
Sometime ago, there was a discussion about slide bookmarking. This would be a better usage for such combinations. And I agree: changing colors with the toolbox seems more appropriate. I doubt that you are remembering the colors behind the combinations so you have no advantage of that feature.

Reply:
Although we both agree changing colors is more convenient using the GUI provided by the toolbox, I cannot exclude some users might have a different opinion. In any case, I believe the action itself - `setPenColor` - should remain, in particular for interaction through DBus.